watch_connectivity_garmin/lib/src/garmin_iq_connect_type.dart

```diff
@@ -0,0 +1,8 @@
+/// Enum for type of Garmin connection to establish
+enum GarminIQConnectType {
```

Comment:
I say just put this in the garmin_initialization_options file

Reply:
Also I'm thinking we should name this `GarminIqConnectionType`. What do you think? I usually like strict camel case.

Reply:
Sounds good.  I made the changes